relayer/query_relayer_data.go

```diff
@@ -0,0 +1,64 @@
+package relayer
+
+import (
+	"fmt"
+	"github.com/dymensionxyz/roller/cmd/consts"
+	"github.com/dymensionxyz/roller/config"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"path/filepath"
+)
+
+func GetRelayerStatus(config config.RollappConfig) string {
```

Comment:
what about the balance check as well?

Reply:
We decided to give it up for now, and only show starting when the channel is being created and active afterwards. If the user has insufficient relayer balance on the hub it will not allow him to start, and the rollapp relayer balance is being loaded upon genesis